Assets/Scenes/TutorialLevel.unity

```diff
@@ -4259,7 +4259,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.81600004, y: 0.026612679}
   m_AnchorMax: {x: 0.97700006, y: 0.1902057}
-  m_AnchoredPosition: {x: -2.082489, y: -0.35250854}
+  m_AnchoredPosition: {x: -2.0824585, y: -0.35250854}
```

Comment:
Same move comment here.

Reply:
Again in the cutscene, `ButtonEnd` this time.